hie-plugin-api/Haskell/Ide/Engine/Cradle.hs

```diff
@@ -14,8 +32,296 @@ import HIE.Bios as BIOS
 -- of the project that may or may not be accurate.
 findLocalCradle :: FilePath -> IO Cradle
 findLocalCradle fp = do
-  -- Get the cabal directory from the cradle
   cradleConf <- BIOS.findCradle fp
   case cradleConf of
-    Just yaml -> BIOS.loadCradle yaml
-    Nothing -> BIOS.loadImplicitCradle fp
\ No newline at end of file
+    Just yaml -> fixCradle <$> BIOS.loadCradle yaml
```

Comment:
`fixCradle` is only called when there is an explicit configuration but calls `isStackCradle` which checks for `Cabal-Helper-Stack`? Seems strange.

Reply:
yeah, I wanted to reuse code. 
`isStackCradle` can also be used by `getProjectGhcVersion`, and here, we know that if it is a stack cradle, it definitely had to be a from Bios cradle. Do you think adding a separate function for `fixCradle` would make more sense?